webdriver/tests/element_send_keys/send_keys.py

```diff
@@ -71,3 +71,29 @@ def test_stale_element(session, inline):
 
     response = element_send_keys(session, element, "foo")
     assert_error(response, "stale element reference")
+
+@pytest.mark.parametrize("value",[
+    [u'\ue018', ";"],
+    [u'\ue019', "="],
+    [u'\ue024', "*"],
+    [u'\ue025', "+"],
+    [u'\ue027', "-"],
+    [u'\ue028', "."],
+    [u'\ue029', "/"],
+    [u'\ue01a', "0"],
+    [u'\ue023', "9"]
+    ])
+def test_normalised_key_value(session, inline, value):
```

Comment:
This test doesn't cover all the normalized keys. Maybe add `printable` to the test name?

Reply:
I could but I don't see value in that. It's only printable because I limited the characters due to timeouts happening in the tests if there are more parametrised values.

Reply:
Non-printable keys won't be testable this way and we might have to use an event listener instead to ensure that the correct key got pressed. As such you cannot just extend the parameterized list.